lib/kubernetes-deploy/kubernetes_resource/network_policy.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class NetworkPolicy < KubernetesResource
+    TIMEOUT = 30.seconds
+
+    def status
+      exists? ? "Created" : "Unknown"
```

Comment:
💩 we just merged #427 which standardizes on "Not Found"

Reply:
No worries, changed once again